src/libraries/sendtohelixhelp.proj

```diff
@@ -91,7 +91,7 @@
     <HelixCommand>$(HelixCommand) /p:LocalPackagesPath="%HELIX_CORRELATION_PAYLOAD%\packages\"</HelixCommand>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(HelixCommand)' == '' and '$(MonoEnvOptions)' != ''">
+  <ItemGroup Condition="'$(HelixCommand)' == '' and '$(MonoEnvOptions)' != '' and '$(TargetOS)' != 'Android'">
```

Comment:
I don't see what value adding this condition brings us since `MonoEnvOptions` should be empty for Android runs, right?